components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -20,10 +20,10 @@
       </p>
       <h3>NIH Award</h3>
       <p>{{ getSparcAwardNumber }}</p>
-      <template v-if="externalPublications.length">
+      <template v-if="primaryPublication">
         <h3>{{ associatedPublications }}</h3>
         <external-pub-link
-          v-for="pub in externalPublications"
+          v-for="pub in primaryPublication"
           :key="pub.doi"
           :publication="pub"
         />
```

Comment:
Since is no longer a list, `v-for` won't work. Also, since `primaryPublication` will always be one object, we can refactor this to be 
<external-pub-link :publication="primaryPublication" />

Reply:
nice